app/models/item_category.rb

```diff
@@ -15,6 +15,6 @@ class ItemCategory < ApplicationRecord
   validates :description, length: { maximum: 250 }
 
   belongs_to :organization
-  has_many :items, -> { order(name: :asc) }, inverse_of: :item_category
+  has_many :items, -> { order(name: :asc) }, inverse_of: :item_category, dependent: :restrict_with_exception
```

Comment:
@dorner (or anyone) what should the behavior be when deleting an ItemCategory for the associated Items? Here I propose that we raise an error, so you can only delete ItemCategory that has no Items.

Reply:
Change to :nullify